src/dynamicTypes/eventHandlersService.ts

```diff
@@ -1,10 +1,29 @@
-import {
```

Comment:
consider having types/interfaces exported via the module/folder they relate to rather than a huge types file